packages/core/test/unit/patchers/aws3_p.test.js

```diff
@@ -205,7 +255,15 @@ describe('AWS v3 patcher', function() {
         await awsClient.send(awsRequest).catch(() => null);
 
         getCauseStub.should.have.been.calledWithExactly(500);
-        closeStub.should.have.been.calledWithExactly(sinon.match({ message: error.message, name: error.code}), true);
+        closeStub.should.have.been.calledWithExactly(sinon.match({ message: error.message, name: error.code }), true);
+      });
+
+      it('should pass params into segment', async function () {
+        await ddbClient.send(ddbGetRequest);
+
+        assert.isTrue(addNewSubsegmentStub.calledWith('DynamoDB'));
+        addNewSubsegmentStub.returnValues[0].should.have.property('aws');
+        addNewSubsegmentStub.returnValues[0].aws.should.include({ consistent_read: true, table_name: 'TestTableName', consumed_capacity: 10 });
```

Comment:
For completeness - [`PutItem` also returns `ItemCollectionMetrics`](https://github.com/aws/aws-xray-sdk-node/blob/e32dd9580def3556ddee50b0090e2c9b97bcf361/packages/core/lib/resources/aws_whitelist.json#L78-L86)

Reply:
I'll leave it as-is, since I wanted to test it with GetItem command but forgot to change the request class name. If you want me to test it with PutItem I can change it, but I think this test is already thorough enough